### PR TITLE
[ODS-5835] Promote Packages workflow promotes with Standard Version nuget packages 

### PR DESCRIPTION
**Pull request by semalaiappan**
Verify all packages under Current packages for Suite 3 v6.1+:
are deployed with prerelease and also verify that it has 4.0.0 as well for many packages based on configuration packages json

https://techdocs.ed-fi.org/pages/viewpage.action?pageId=58721615

you can find packages list here https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/runs/5017187991/jobs/8994999884#step:5:123

### CODE REVIEW

File: logistics/scripts/activities/build/promote-packages.ps1 [hunk 1]
```diff
@@ -42,6 +45,21 @@ Invoke-Task -name 'Gather package versions' -task {
         (Select-ConfigurationPackages $configurationFile)
     )
 
+    $keysToRemove = @()
+
```
Comment (semalaiappan) [inline logistics/scripts/activities/build/promote-packages.ps1:49]: This code is remove the unwanted package like here edfi.suite3.ods.standard & edfi.suite3.odsapi.testsdk
$packages = @{
  "edfi.suite3.ods.standard" = "1.0.0"
  "edfi.suite3.ods.standard.4.0.0" = "7.0.12"
  "edfi.suite3.odsapi.testsdk.4.0.0" = "latest"
  "edfi.suite3.odsapi.testsdk" = "7.0.921"
}

File: configuration.packages.json [hunk 2]
```diff
@@ -71,22 +71,22 @@
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Admin.PostgreSQL": {
-      "PackageName": "EdFi.Database.Admin.PostgreSQL",
+      "PackageName": "EdFi.Database.Admin.PostgreSQL.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:74]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 3]
```diff
@@ -71,22 +71,22 @@
...
       "PackageVersion": "latest",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Admin": {
-      "PackageName": "EdFi.Database.Admin",
+      "PackageName": "EdFi.Database.Admin.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:79]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 4]
```diff
@@ -71,22 +71,22 @@
...
       "PackageVersion": "latest",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Security": {
-      "PackageName": "EdFi.Database.Security",
+      "PackageName": "EdFi.Database.Security.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:84]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 5]
```diff
@@ -71,22 +71,22 @@
...
       "PackageVersion": "latest",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Database.Security.PostgreSQL": {
-      "PackageName": "EdFi.Database.Security.PostgreSQL",
+      "PackageName": "EdFi.Database.Security.PostgreSQL.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:89]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 6]
```diff
@@ -111,12 +111,12 @@
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.OdsApi.Sdk": {
-      "PackageName": "EdFi.Suite3.OdsApi.Sdk",
+      "PackageName": "EdFi.Suite3.OdsApi.Sdk.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:114]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 7]
```diff
@@ -111,12 +111,12 @@
...
       "PackageVersion": "latest",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.OdsApi.TestSdk": {
-      "PackageName": "EdFi.Suite3.OdsApi.TestSdk",
+      "PackageName": "EdFi.Suite3.OdsApi.TestSdk.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:119]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 8]
```diff
@@ -141,12 +141,12 @@
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Ods.WebApi": {
-      "PackageName": "EdFi.Suite3.Ods.WebApi",
+      "PackageName": "EdFi.Suite3.Ods.WebApi.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:144]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 9]
```diff
@@ -141,12 +141,12 @@
...
       "PackageVersion": "latest",
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.RestApi.Databases": {
-      "PackageName": "EdFi.Suite3.RestApi.Databases",
+      "PackageName": "EdFi.Suite3.RestApi.Databases.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:149]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: configuration.packages.json [hunk 10]
```diff
@@ -166,7 +166,7 @@
       "PackageSource": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
     },
     "EdFi.Ods.Standard": {
-      "PackageName": "EdFi.Suite3.Ods.Standard",
+      "PackageName": "EdFi.Suite3.Ods.Standard.{StandardVersion}",
```
Comment (simpat-jesus) [inline configuration.packages.json:169]: PackageName is missing 'Standard.' before '{StandardVersion}'

File: logistics/scripts/activities/build/promote-packages.ps1 (hunk 1 above)
Comment (simpat-jesus) [reply logistics/scripts/activities/build/promote-packages.ps1:49]: as per https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/runs/5017187991/jobs/8994999884#step:5:123
 packages are filtered out, ant not published

**Review by simpat-jesus — CHANGES_REQUESTED** [review]

**Comment by semalaiappan** [comment]
@simpat-jesus  ackageName is missing 'Standard.' before '{StandardVersion}' - Fixed it in new commit . Ready for review

**Review by simpat-jesus — APPROVED** [review]

**Pull request merged**